src/fsharp/FSharp.Core/string.fsi

```diff
@@ -18,6 +18,29 @@ namespace Microsoft.FSharp.Core
     [<RequireQualifiedAccess>]
     module String = 
 
+        /// <summary>Builds a new string whose characters are the results of applying the function <c>mapping</c>
+        /// to each of the characters of the input string and concatenating the resulting
+        /// strings.</summary>
+        ///
+        /// <param name="mapping">The function to produce a string from each character of the input string.</param>
+        /// <param name="str">The input string.</param>
+        ///
+        /// <returns>The concatenated string.</returns>
+        /// 
+        /// <example> The following samples shows how to interspace spaces in a text
+        /// <code lang="fsharp">
+        /// String.collect (sprintf "%c ") "Stefan says: Hi!"  // evaluates "S t e f a n   s a y s :   H i ! "
```

Comment:
Use pipelining for each example, and suggest binding the input to make the example stand out more prominently:
```fsharp
        /// let inp = "Stefan says: Hi!"
        /// inp |> String.collect (sprintf "%c ") // evaluates "S t e f a n   s a y s :   H i ! "
```